resources/config/laravel-sitemap.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+
+  // Settings for GuzzleHttp\Client
```

Comment:
Wrap all these in another key called `guzzle_options`. Instead of the current comment explain in full sentences how these options will be used